docs/word/annotations.md

```diff
@@ -0,0 +1,363 @@
+  - [AnnotationCollection](/javascript/api/word/word.annotationcollection): Represents the collection of annotations.
+  - [AnnotationSet](/javascript/api/word/word.annotationset): Represents the set of annotations produced by your add-in in this session.
+  - [CritiqueAnnotation](/javascript/api/word/word.critiqueannotation): Represents the critique type of annotation.
+  - [Critique](/javascript/api/word/word.critique): Represents feedback about an affected area of a paragraph, indicated by a colored underline.
+- Annotation events on [Document](/javascript/api/word/word.document):
+  - [onAnnotationClicked](/javascript/api/word/word.document#word-word-document-onannotationclicked-member)
+  - [onAnnotationHovered](/javascript/api/word/word.document#word-word-document-onannotationhovered-member)
+  - [onAnnotationInserted](/javascript/api/word/word.document#word-word-document-onannotationinserted-member)
+  - [onAnnotationPopupAction](/javascript/api/word/word.document#word-word-document-onannotationpopupaction-member)
+  - [onAnnotationRemoved](/javascript/api/word/word.document#word-word-document-onannotationremoved-member)
+
+## Use annotation APIs
+
+The following sections show how to work with annotation APIs. These examples are based on the [Manage annotations](https://github.com/OfficeDev/office-js-snippets/blob/prod/samples/word/50-document/manage-annotations.yaml) code sample.
+
+In your add-in, your code should manage annotations more dynamically according to feedback or critique results from your service run against the user's content in the document.
+
+## Register annotation events
+
+The following code shows how to register event handlers. To learn more about working with events in Word, see [Work with events using the Word JavaScript API](word-add-ins-events.md). See the following sections for example annotation event handlers.
+
+```typescript
+let eventContexts = [];
+
+async function registerEventHandlers() {
+  // Registers event handlers.
+  await Word.run(async (context) => {
+    eventContexts[0] = context.document.onParagraphAdded.add(paragraphChanged);
+    eventContexts[1] = context.document.onParagraphChanged.add(paragraphChanged);
+
+    eventContexts[2] = context.document.onAnnotationClicked.add(onClickedHandler);
+    eventContexts[3] = context.document.onAnnotationHovered.add(onHoveredHandler);
+    eventContexts[4] = context.document.onAnnotationInserted.add(onInsertedHandler);
+    eventContexts[5] = context.document.onAnnotationRemoved.add(onRemovedHandler);
+    eventContexts[6] = context.document.onAnnotationPopupAction.add(onPopupActionHandler);
+
+    await context.sync();
+
+    console.log("Event handlers registered.");
+  });
+}
+```
+
+### onClickedHandler event handler
+
+The following code runs when the registered `onAnnotationClicked` event occurs.
+
+```typescript
+async function onClickedHandler(args: Word.AnnotationClickedEventArgs) {
+  await Word.run(async (context) => {
```

Comment:
For consistency, do you want to add a remark here and in the next four samples?

Reply:
Added